deploy_tools/cli.py

```diff
@@ -97,6 +97,14 @@ def validate_address(ctx, param, value):
     help="Toggles the solidity optimizer",
     envvar="OPTIMIZE",
 )
+optimize_runs_option = click.option(
+    "--optimize-runs",
+    default=500,
+    type=int,
+    show_default=True,
+    help="Number of optimization runs when compile contracts",
```

Comment:
This text is a little misleading. It sounds like it is the setting of how often the compiler will run the optimize algorithm. But this number is actually for how many contract runs the optimizer will optimize. 
So change it to
`Number of contract runs to optimize for when compiling contracts`

Reply:
When I read it again with your comment if see you point. Thanks! :smiley:

Reply:
:heavy_check_mark: